src/layouts/navbar/NavBar.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import "./navbar.css";
+import { Link } from "react-router-dom";
+function NavBar() {
+  const [isOpen, setIsOpen] = useState(true);
+  const toggleIsOpen = () => {
+    setIsOpen((prev) => !prev);
+  };
+  return (
+    <header className="navbar-container bg-primary">
+      <Link to={"/"} className="link white">
+        <h2 className="nav-logo">Emporium</h2>
+      </Link>
+
+      <nav>
+        <ul className={"list nav-links-container " +  (!isOpen ? "hidden" : "")}>
```

Comment:
I noticed that if I close the list on a small screen it will also disappear on bigger screens as well which shouldn't happen.
![issue](https://user-images.githubusercontent.com/85923879/158774276-4cf217f1-b036-4ef4-92ee-4cbf37d99b16.png)

Reply:
Thanks for review, Fixed it.